tutorials/e2e-huggingface-summarization/src/run_summarization/baseline.py

```diff
@@ -0,0 +1,62 @@
+import nltk
+from nltk.tokenize import sent_tokenize
+from datasets import load_dataset, load_metric
+import os
+import argparse
+import logging
+from azureml.core import Run
+import mlflow
+
+
+logger = logging.getLogger(__name__)
+nltk.download("punkt")
+rouge_score = load_metric("rouge")
+
+def three_sentence_summary(text):
+    return "\n".join(sent_tokenize(text)[:3])
+
+
+def evaluate_baseline(datasets, metric, text_column, summary_column):
```

Comment:
could you add types and docstrings to these functions?

Reply:
done